pandas/tests/frame/test_quantile.py

```diff
@@ -155,6 +143,28 @@ def test_quantile_interpolation(self):
                              index=[.25, .5], columns=['a', 'b', 'c'])
         assert_frame_equal(result, expected)
 
+    def test_quantile_interpolation_datetime(self, datetime_frame):
+        # see gh-10174
+
+        # interpolation = linear (default case)
+        df = datetime_frame
+        q = df.quantile(0.1, axis=0, interpolation='linear')
+        assert q['A'] == np.percentile(df['A'], 10)
+
+    def test_quantile_interpolation_int(self, int_frame):
+        # see gh-10174
+
+        df = int_frame
+        # interpolation = linear (default case)
+        q = df.quantile(0.1)
+        assert q['A'] == np.percentile(df['A'], 10)
+
+        # TODO: Remove the following, as q1 is not different from q
```

Comment:
sorry to be pedantic here, can you change this to 

`# TODO: q1 is not different from q`

and move it below 

`# test with and without interpolation keyword`